src/app_service/service/community/service.nim

```diff
@@ -769,7 +769,8 @@ QtObject:
 
       updatedCommunity.settings = communitySettings
       self.communities[communityId] = updatedCommunity
-      self.chatService.loadChats()
+      # TODO this might be broken. Can we fetch only the community instead?
```

Comment:
Still valid?

Reply:
Yes. I'll fix it along with the other bug I found. Expect a small to medium commit coming in today